app/services/bill-runs/two-part-tariff/submit-review-bill-run.service.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+/**
+ * Updates the session cookie with the filter data needed for the two-part tariff review bill run page
+ * @module SubmitReviewBillRunService
+ */
+
+/**
+ * Updates the session cookie with the filter data needed for the two-part tariff review bill run page
+ *
+ * @param {Object} payload The `request.payload` containing the filter data.
+ * @param {Object} yar - The Hapi `request.yar` session manager passed on by the controller
+ */
+async function go (payload, yar) {
+  const clearFilters = payload?.clearFilters
+  const filterIssues = payload?.filterIssues
+  const filterLicenceHolder = payload?.filterLicenceHolder
+  const filterLicenceStatus = payload?.filterLicenceStatus
+
+  if (clearFilters) {
+    yar.clear('filters')
+  } else {
+    yar.set('filters', {
```

Comment:
As per my previous suggestion to avoid any possible issues should we add filters elsewhere in the service.

```suggestion
    yar.clear('reviewFilters')
  } else {
    yar.set('reviewFilters', {
```